mozilla/addons-server
# save appeal text to a model and expose in reviewer tools

Fixes: mozilla/addons#14877



### Description



### Context

We've collected the appeal text and submitted it to Cinder for appeals handled there for many months, but that we didn't save it ourselves too was seeming overlooked.  We need this information in the reviewer tools so reviewers who are handling appeals there have the full context.  Reviewer tools appeals happen when the original decision was made in the reviewer tools (e.g. the abuse report was directed to the reviewer tools originally; it was escalated; or it was a proactive add-on review)

### Testing

It's a little involved to set-up properly:
1) set `dsa-appeals-review` waffle switch to enabled
1) make a decision in the reviewer tools on an add-on version
1) pull the appeal link from the email in FakeMail in django admin
1) using the appeal link, fill out the form with some appeal text
1) go back to the reviewer tools (the review should be back in the queue with a due_date)
1) see the appeal listed as an appeal job
1) in the expanded details for the appeal job see the appeal text you entered earlier.

(If already have an appeal there from before you can just create the `CinderAppealText` object directly in the reviewer tools and jump to the last step)

### Checklist




![Screenshot 2024-07-02 033126](https://github.com/mozilla/addons-server/assets/768592/9066d687-c759-49ae-81b4-fc11b066d5ec)
